shared/src/business/entities/cases/PublicDocketEntry.test.js

```diff
@@ -19,6 +19,16 @@ describe('PublicDocketEntry', () => {
       processingStatus: 'testing',
       receivedAt: 'testing',
       servedAt: '2019-03-01T21:40:46.415Z',
+      servedParties: [
```

Comment:
Since this field was removed in the entity definition, does this field actually need to be defined here?

Reply:
This test verifies that `servedParties` doesn't get included in the instantiated `PublicDocketEntry`.